### PR TITLE
Merge test job updates from master

### PR DESCRIPTION
**Pull request by mmitche**
Also update the test job under buildpipeline\tests\ to match the old version under buildpipeline\.  There were missing SCEP steps.

### CODE REVIEW

**Comment by mmitche** [comment]
@jcagme PTAL

File: buildpipeline/tests/Dotnet-CoreClr-Trusted-BuildTests.json [hunk 1]
```diff
@@ -109,6 +109,82 @@
         "failOnStandardError": "false"
       }
     },
+    {
```
Comment (jcagme) [inline buildpipeline/tests/Dotnet-CoreClr-Trusted-BuildTests.json]: SCEP ops was generalized and we use a script on top of all "extension tasks". This file already has these two (first and last tasks) so you don't need to add any SCEP tasks anymore

Comment (mmitche) [reply buildpipeline/tests/Dotnet-CoreClr-Trusted-BuildTests.json]: @jcagme So the changes to this BuildTests (aside from the queues) are uneccessary?  If so, I saw some new access denied errors last night.

Comment (jcagme) [reply buildpipeline/tests/Dotnet-CoreClr-Trusted-BuildTests.json]: Correct. There access denied errors are related to the agent not recognizing the defined user capability, hence SCEP is not being disable.

**Comment by mmitche** [comment]
@jcagme Thanks, removed

**Pull request merged**